### DISCUSSION TITLE
Payer plans

### DISCUSSION THREAD

**Discussion by sklukasatphs** [CMSgov/hospital-price-transparency#91] [Q&A] [answered]
One of our leaders has concerns about plan level data.  We have some payer plans from prior years that are kept open for old claims.  They would not apply for patient visits during this year.  Do we need to add those payer plans to the MRF?

**Comment by vduffel99** [comment, marked as answer]
The 45CFR 180.20 defines:
Payer-specific negotiated charge as the charge that a hospital has negotiated with a third party payer for an item or service AND

Third party payer is an entity that is, by statute, contract, or agreement, legally responsible for payment of a claim for a healthcare item or service

If there is a contract (legal responsibility for payment), they should be listed in the MRF.

**Reply by RidenourK** [reply]
@sklukasatphs The above user is correct.
